airbyte-integrations/connectors/source-xkcd/integration_tests/acceptance.py

```diff
@@ -10,4 +11,6 @@
 @pytest.fixture(scope="session", autouse=True)
```

Comment:
@btkcodedev just delete the whole file.

In fact, if there are NO python custom components, can you delete all python files, and move `manifest.yaml` to top level? Just leave the following:
- `metadata.yaml`
- `manifest.yaml`
- docs
- README (we'll need to rewrite it)